ui/js/components/perf/compare.js

```diff
@@ -108,3 +108,52 @@ treeherder.component('compareError', {
         newRevision: '<'
     }
 });
+
+treeherder.component('distributionGraph', {
+    template: '<canvas id="a" width="400" height="30"></canvas>',
+    bindings: {
+        replicates: '<'
+    },
+    controller: [function () {
+        var ctrl = this;
```

Comment:
We don't enforce this globally in treeherder yet, but for new code it would be good to use `const` or `let` instead of `var` for declaring variables:

http://airbnb.io/javascript/#references--prefer-const